stubs/openssl-python/OpenSSL/crypto.pyi

```diff
@@ -1,197 +1,220 @@
 import sys
 from datetime import datetime
-from typing import Callable, Iterable, List, Optional, Set, Text, Tuple, Union
+from typing import Callable, Iterable, List, Optional, Sequence, Set, Text, Tuple, Union
 
-from cryptography.hazmat.primitives.asymmetric import dsa, rsa
+from cryptography.hazmat.primitives.asymmetric.dsa import DSAPrivateKey, DSAPublicKey
+from cryptography.hazmat.primitives.asymmetric.rsa import RSAPrivateKey, RSAPublicKey
+from cryptography.x509 import Certificate, CertificateRevocationList, CertificateSigningRequest
+
+if sys.version_info >= (3, 0):
+    _BytesLike = bytes
```

Comment:
Remove in favor of just using `bytes`. `bytes = str` on Python 2.

Reply:
Fixed.